requirements/requirements.txt

```diff
@@ -1,3 +0,0 @@
-django>=1.9.6
-requests
-python-jose>=1.3.2
```

Comment:
This is redundant with what's in `setup.py`. Seems better to just `pip install -e .` rather than maintain two copies of things. This reduces the likelihood that they become out of sync again.